src/components/CompareResults/CompareResultsView.tsx

```diff
@@ -24,16 +26,27 @@ function CompareResultsView(props: CompareResultsViewProps) {
     const searchParams = new URLSearchParams(location.search);
     const repos = searchParams.get('repos')?.split(',');
     const revs = searchParams.get('revs')?.split(',');
-    void dispatchFetchCompareResults(repos as Repository['name'][], revs);
-  });
+    async function fetchCompareResults() {
+      await dispatchFetchCompareResults(repos as Repository['name'][], revs);
+    }
+    void fetchCompareResults();
+
+    // Clear the checked revisions when the revisions prop changes
+    dispatchSetCheckedRevisions([]);
+  }, [
+    revisions,
+    location.search,
+    dispatchFetchCompareResults,
+    dispatchSetCheckedRevisions,
+  ]);
 
   return (
-    <Container maxWidth="xl">
+    <Container maxWidth='xl'>
```

Comment:
Can you change these quotes to double quotes, as they were in the original code?

Reply:
Okay...I missed that..making the change.